### PR TITLE
chore: Codification of Code (WIP)

### LINKED ISSUE #7312: [ENH] Codification of `keptn` Code
`keptn` is having low score on [goreportcard](https://goreportcard.com/report/github.com/keptn/keptn#golint). As per this, it is needed to enhance the code to make it as much as lint error free code. 

**Environment**

* Client OS (e.g., Linux, macOS, Windows):
* Keptn Version (`keptn version`):
* Kubernetes Cloud Provider (e.g., GKE, AKS):
* Kubernetes version (`kubectl version`):

### PR DESCRIPTION
**Pull request by jainammm**
## This PR

- [x] Fix gofmt
- [ ] Fix ineffassign
- [ ] Fix golint
- [ ] Fix gocyclo

### Related Issues

Fixes #7312

### CODE REVIEW

**Comment by jainammm** [comment]
So far added fix for `gofmt`... Ran this command to fix it `gofmt -s -w .`

**Comment by afzal442** [comment]
Sorry @jainammm, I don't understand, why we need to improve other than `golint`. Plz revert the changes, and we can focus on  lint issues, you can reformat as per the requirement.
> to fix golint, we'll have to either address all comments or configure golint to exclude certain things like uncommented exported functions

https://goreportcard.com/report/github.com/keptn/keptn#golint

**Comment by jainammm** [comment]
Hey @afzal442, yes sure I can revert back the changes... But the fix for `gofmt` was a simple one... So I did it but yes we can keep this limited to `golint`

**Comment by jainammm** [comment]
Hey @afzal442, the pipeline failed.

1. In `Unit tests Microservices`: Test `helm-service` failed... The error says that it is redeclared... But I've only changed the name

File: bridge/client/app/_components/ktb-create-service/ktb-create-service.component.spec.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ import { AppModule } from '../../app.module';
 import { NotificationsService } from '../../_services/notifications.service';
 import { NotificationType } from '../../_models/notification';
 import { HttpErrorResponse } from '@angular/common/http';
-import { ApiService } from '../../_services/api.service';
+import { APIService } from '../../_services/api.service';
```
Comment (afzal442) [inline bridge/client/app/_components/ktb-create-service/ktb-create-service.component.spec.ts:10]: I think, this is an exported class name, why do we need this change? It's unrelated to `golint` warnings.

File: distributor/pkg/api/internal.go [hunk 2]
```diff
@@ -35,7 +36,7 @@ type InternalService int
 const (
 	ConfigurationService InternalService = iota
 	ShipyardController
-	ApiService
+	APIService
```
Comment (afzal442) [inline distributor/pkg/api/internal.go:39]: It won't affect frontend, Ig.

File: cli/cmd/generate_docs.go [hunk 3]
```diff
@@ -50,7 +49,7 @@ keptn generate docs --dir=/some/directory`,
 		// check if output directory exists
 		if _, err := os.Stat(outputDir); os.IsNotExist(err) {
 			// outputDir does not exist
-			return errors.New(fmt.Sprintf("Error trying to access directory %s. Please make sure the directory exists.", outputDir))
+			return fmt.Errorf("Error trying to access directory %s. Please make sure the directory exists", outputDir)
```
Comment (afzal442) [inline cli/cmd/generate_docs.go:52]: test case failing here, it should reflect it changes to its test file as well. e.g. cli/cmd/generate_docs_test.go

File: helm-service/pkg/apis/networking/istio/v1alpha3/destination_rule.go [hunk 4]
```diff
@@ -355,32 +355,32 @@ type LoadBalancerSettings_ConsistentHashLB struct {
...
-	UseSourceIp bool `protobuf:"varint,3,opt,name=use_source_ip,json=useSourceIp,proto3,oneof"`
+type LoadBalancerSettingsConsistentHashLBUseSourceIP struct {
+	UseSourceIP bool `protobuf:"varint,3,opt,name=use_source_ip,json=useSourceIp,proto3,oneof"`
 }
 
 // Describes a HTTP cookie that will be used as the hash key for the
 // Consistent Hash load balancer. If the cookie is not present, it will
 // be generated.
-type LoadBalancerSettings_ConsistentHashLB_HTTPCookie struct {
+type LoadBalancerSettingsConsistentHashLBHTTPCookie struct {
```
Comment (afzal442) [inline helm-service/pkg/apis/networking/istio/v1alpha3/destination_rule.go:377]: See line 367 and compare it. It's re declared.

**Comment by afzal442** [comment]
hey @jainammm, let me know if you want to work further. Plz resolve the comments. Thanks.

**Comment by jainammm** [comment]
Hey @mowies 
There were some complications in fixing the linting... Also I was not sure if it would affect code at other places... So, not really sure what to do with this PR... We can close it i guess

**Pull request closed without merging**